conf/airframes/examples/bebop.xml

```diff
@@ -10,6 +10,17 @@
       <module name="udp"/>
     </target>
 
+    <define name="MT9F002_OUTPUT_HEIGHT" value="640" />
```

Comment:
We are working on a separate vision library which will make the paparazzi vision implementation obsolete.